pkg/util/utils.go

```diff
@@ -82,7 +82,7 @@ func NormalizeName(name string) string {
 }
 
 func normalizeNameByLimit(name string, suffix string, limit int) string {
```

Comment:
This function is also called by `NormalizeName` and `NormalizeLabelKey` which are finally called by `updateExpressionsMatchLabels`, could you double check if the change has any impact on that function?

Reply:
Yes, there is no impact to updateExpressionsMatchLabels. If there is label change, the controller will sync the new generated labels to nsx tags, so the resource will be updated.